resources/osl_matrix.rb

```diff
@@ -0,0 +1,92 @@
+resource_name :osl_matrix
```

Comment:
Let's go ahead and rename this as `osl_synapse` since that's what we're actually deploying. The cookbook deals with all things Matrix and we may end up deploying another server down the road so this will keep it separate.